feature-group-definitions/ic.dist.yml

```diff
@@ -0,0 +1,17 @@
+# Generated from: ic.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: ic unit
+description: The CSS `ic` unit corresponds to the width of CJK ideographic characters.
+spec: https://drafts.csswg.org/css-values-4/#ic
+status:
+  baseline: false
```

Comment:
This isn't baseline because of the `CSS.ic()` static method not being implemented in Firefox:
https://developer.mozilla.org/en-US/docs/Web/API/CSS#browser_compatibility

But this isn't really preventing its use in a vast majority of cases. I think we should remove api.CSS.ic_static from this feature or use an editorial override.